nengo/builder/operator.py

```diff
@@ -643,6 +643,70 @@ def step_dotinc():
         return step_dotinc
 
 
+class BsrDotInc(Operator):
```

Comment:
Oh god, you're totally right. In that case, since making an offline note of whether a warning has been given recently would be way too painful, I'm totally fine with just a silent warning.